src/main/java/duke/command/Add.java

```diff
@@ -0,0 +1,175 @@
+package duke.command;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import duke.TaskList;
+import duke.exception.CommandNotDefinedException;
+import duke.exception.DukeException;
+import duke.exception.EmptyTextException;
+import duke.exception.TimeFormatException;
+import duke.exception.TimeInconsistException;
+import duke.exception.WrongUsageException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Class for adding a task into task list
+ */
+public class Add implements Command {
```

Comment:
Would a class name such as AddTask be clearer in this context?

Reply:
I agree with this comment here. It may make it even clearer if you call it `AddTaskCommand`, say.